tests/runtime/usdt

```diff
@@ -152,6 +152,11 @@ TIMEOUT 5
 BEFORE ./testprogs/usdt_test
 REQUIRES ./testprogs/usdt_test should_not_skip
 
+NAME "usdt probes - attach to multiple probes with different number of locations by wildcard"
+RUN {{BPFTRACE}} -e 'usdt:./testprogs/usdt_multiple_locations:tracetest:testprobe* { printf("here\n" ); exit(); }' -c ./testprogs/usdt_multiple_locations
+EXPECT Attaching 6 probes...
+TIMEOUT 5
```

Comment:
```suggestion
TIMEOUT 5
REQUIRES ./testprogs/usdt_multiple_locations should_not_skip
```
This is necessary so that the check for SystemTap headers in `main` works.